skyplane/obj_store/gcs_interface.py

```diff
@@ -193,7 +194,7 @@ def upload_object(self, src_file_path, dst_object_name, part_number=None, upload
 
         if part_number is None:
             blob = bucket.blob(dst_object_name)
-            blob.upload_from_filename(src_file_path)
+            blob.upload_from_filename(src_file_path, content_type=mime_type or "application/octet-stream")
```

Comment:
what's the reason for setting the default to "application/octet-stream"?

Reply:
It defaults to that on GCP, but I left it as None now since GCP will automatically substitute the content_type if not set https://cloud.google.com/storage/docs/metadata#content-type